src/applications/static-pages/ivc-champva/10-10D/entry.js

```diff
@@ -6,7 +6,7 @@ import { Provider } from 'react-redux';
 export default function create1010DAccess(store, widgetType) {
   const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
   if (root) {
-    import(/* webpackChunkName: "form-10-10d" */ './App').then(module => {
+    import(/* webpackChunkName: "form1010d" */ './App').then(module => {
```

Comment:
This ID has to match the ID in `vets-website/src/applications/static-pages/widgetTypes.js`